masterlist.yaml

```
@@ -9547,6 +9547,40 @@ plugins:
         util: '[SSEEdit v4.0.2](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         udr: 24
 
+  - name: 'DWKaPoTunRace.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/40928/'
+        name: 'Races of Nirn - Kapo''tun SE'
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/77612/'
+        name: 'Races of Nirn - Kapotun Reborn'
+    inc:
+      - name: 'Kapotun SOS.esp'
+        condition: 'version("DWKaPoTunRace.esp", "2.0.0", >=)'
+      - name: 'Kapotun Vanilla Armor Patch.esp'
+        condition: 'version("DWKaPoTunRace.esp", "2.0.0", >=)'
+    msg:
+      - <<: *obsolete
+        subs: [ '[Races of Nirn - Kapotun Reborn](https://www.nexusmods.com/skyrimspecialedition/mods/77612/)' ]
+        condition: 'version("DWKaPoTunRace.esp", "2.0", <) and not checksum("DWKaPoTunRace.esp", 672EB2D2)'
+    clean:
+      # Original
+      - crc: 0x672EB2D2
+        util: 'SSEEdit v4.0.4'
+      # Reborn
+      - crc: 0x287A2A43
+        util: 'SSEEdit v4.0.4'
+  - name: 'SOS - Kapotun Addon.esp'
+    clean:
+      - crc: 0xFCE78411
+        util: 'SSEEdit v4.0.4'
+  - name: 'Kapotun ARMA .+\.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/77612/'
+        name: 'Races of Nirn - Kapotun Reborn'
+    inc:
+      - name: 'DWKaPoTunRace.esp'
+        condition: 'version("DWKaPoTunRace.esp", "2.0.0", >=)'
```

These should be sorted the other way around:
```suggestion
  - name: 'Kapotun ARMA .+\.esp'
    url:
      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/77612/'
        name: 'Races of Nirn - Kapotun Reborn'
    inc:
      - name: 'DWKaPoTunRace.esp'
        condition: 'version("DWKaPoTunRace.esp", "2.0", >=)'
  - name: 'SOS - Kapotun Addon.esp'
    clean:
      - crc: 0xFCE78411
        util: 'SSEEdit v4.0.4'
```